package.json

```diff
@@ -88,7 +88,7 @@
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom --colors",
-    "test:update": "npm run test -- --updateSnapshot",
+    "test:coverage": "npm run test -- --coverage",
     "test:ci": "CI=true npm run test -- --coverage --runInBand --colors",
```

Comment:
Unrelated, but we didn't need an `update` script (just hit `u` when the tests have run and are waiting for changes).

Coverage reports are in `coverage/lcov-report/index.html`